login.go

```diff
@@ -32,7 +34,8 @@ func (c *LoginController) Authorize(ctx *app.AuthorizeLoginContext) error {
 // Generate runs the authorize action.
 func (c *LoginController) Generate(ctx *app.GenerateLoginContext) error {
 	if !configuration.IsPostgresDeveloperModeEnabled() {
-		return ctx.Unauthorized()
+		jerrors, _ := jsonapi.ConvertErrorFromModelToJSONAPIErrors(models.NewUnauthorizedError("Postgres developer mode not enabled"))
```

Comment:
You could certainly do that but it takes much more effort. How to generate the `id` field for instance?